src/index.js

```diff
@@ -120,6 +121,7 @@ octokit
 async function start() {
   // validate key first
   if (shouldValidate) {
+    core.debug(`Event JSON: ${JSON.stringify(eventJSON)}`);
```

Comment:
@weineran I think we should leave logging the full eventJSON out of the action. Although unlikely, GitHub _could_ add sensitive information to the eventJSON (hopefully they would redact any secrets, but you never know). Logging the whole context just gives me a bit of anxiety, it's kinda like logging `env`. WDYT?

Reply:
Good thinkin!